pypeit/scripts/setup_coadd2d.py

```diff
@@ -43,8 +44,19 @@ def get_parser(cls, width=None):
                                  'use --det 1 5; to coadd mosaics made up of detectors 1,5 and '
                                  '3,7, you would use --det 1,5 3,7')
         parser.add_argument('--only_slits', type=str, nargs='+',
-                            help='A space-separated set of slits to coadd.  If not provided, all '
-                                 'slits are coadded.')
+                            help='A space-separated set of slits to coadd. Example syntax -- '
+                                 '--only_slits DET01:175,DET02:205 or MSC02:2234.  If not provided, all '
+                                 'slits are coadded. If both --det and --only_slits are provided, '
+                                 '--det will be ignored. This and --exclude_slits are mutually exclusive. '
+                                 'If both are provided, --only_slits takes precedence.')
+        parser.add_argument('--exclude_slits', type=str, nargs='+',
+                            help='A space-separated set of slits to exclude in the coaddition. '
+                                 'This and --only_slits are mutually exclusive. '
+                                 'If both are provided, --only_slits takes precedence.')
+        parser.add_argument('--spat_toler', type=int, default=None,
```

Comment:
Shuuldn't this have a default, and I'm not sure why it is an integer. It seems it should be a float.